onyo/lib/ui.py

```diff
@@ -26,14 +26,14 @@ class UI(object):
 
     Attributes
     ----------
-    log: Logger
+    logger
```

Comment:
This should be "log", not "logger", based on the variable in line 9 :)

Reply:
Interesting about line 9. That seems to be unused in the rest of the file.

But this indeed should be `logger`.  `log()` is a function, and `self.logger` is an attribute of the `UI` class. And everything uses that.